app/javascript/src/components/Dashboard/Notes/List/FilterBar.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+import { Typography } from "neetoui/v2";
+import { MenuBar } from "neetoui/v2/layouts";
+
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
```

Comment:
Please use aliases here.

Reply:
@karthiknmenon Can you give me an example of the aliases here?

Reply:
Instead of `@bigbinary/neeto-icons`. We could use `neeto-icons`. 
This is a standard practice that we are following with the revamp in neeto products.